packages/core/src/components/hotkeys/hotkeysDialog.tsx

```diff
@@ -30,7 +30,8 @@ class HotkeysDialog {
     private container: HTMLElement;
     private hotkeysQueue = [] as IHotkeyProps[][];
     private isDialogShowing = false;
-    private timeoutToken = 0;
+    private showTimeoutToken = 0;
+    private hideTimeoutToken = 0;
```

Comment:
feels more accurate to declare these as `number` rather than initialize to a likely invalid ID (although `clearTimeout` happily ignores invalid IDs)